extern-sdk/python/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
+  - ddysher
+  - gaocegege
+  - codeflitting
+  - ZhuYuJin
```

Comment:
@ZhuYuJin is not the member of Caicloud thus it does not actually work now. When we move the project to public org, it then should work.